### PR TITLE
Update facility admin Gherkin scenarios with 0.13 changes

### PR DESCRIPTION
**Pull request by radinamatic**
### Summary
Ready for review!

Updated according to 0.13 changes, minus 2 that relate channel management workflows (since we are not testing those with b4, TBF(inished) after super admin scenarios are merged).

### Reviewer guidance
Check if something is missing, if the changes are clear or they need rewording.

### References
…

----

### Contributor Checklist

PR process:

- [X] PR has the correct target branch and milestone
- [X] PR has 'needs review' or 'work-in-progress' label
- [X] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: integration_testing/features/admin/admin-change-profile-information.feature [hunk 1]
```diff
@@ -2,17 +2,28 @@ Feature: Admin changes profile information
   Admin needs to be able to change their own profile information
 
   Background:
-    Given I am signed in to Kolibri as an admin user
+    Given I am signed in to Kolibri as a facility admin
       And I am on the *User > Profile* page
 
   Scenario: Admin changes username and full name
-     When I change my full name
+    When I clic the *Edit* button
```
Comment (mrpau-richard) [inline integration_testing/features/admin/admin-change-profile-information.feature]: typo error `clic`

File: integration_testing/features/admin/admin-change-profile-information.feature [hunk 2]
```diff
@@ -2,17 +2,28 @@ Feature: Admin changes profile information
...
+    When I change my full name
       And I change my username
       And if my changes are valid (I did not leave the fields empty)
-      And I click the “Save changes” button
-     Then I see the *Profile details updated* inline notification
-      And I see the new full name and username on the profile page
-      And I see the new username in the user menu
+      And I click the “Save” button
+    Then I see the *Profile details updated* snackbar notification
+    When I go back to my *Profile > Detils* page
```
Comment (mrpau-richard) [inline integration_testing/features/admin/admin-change-profile-information.feature]: typo error `Detils `

File: integration_testing/features/admin/admin-change-profile-information.feature [hunk 3]
```diff
@@ -2,17 +2,28 @@ Feature: Admin changes profile information
...
+    Then I see the new full name and username on the profile page
+
+  Scenario: Admin selects gender and birth year
+    When I clic the *Edit* button
```
Comment (mrpau-richard) [inline integration_testing/features/admin/admin-change-profile-information.feature]: typo `clic `

File: integration_testing/features/admin/admin-find-user.feature [hunk 4]
```diff
@@ -16,7 +16,7 @@ Feature: Admin find users
     When I click the left arrow (previous page) button
     Then I am back on the first page of the list of facility users
     When I keep clicking the right arrow to find a user
-    Then I see the numbers of pages correctly increasing in increments of 10
+    Then I see the numbers of pages correctly increasing in increments of 10 # or 30, if th enumber of users is close to 1000
```
Comment (mrpau-richard) [inline integration_testing/features/admin/admin-find-user.feature:19]: typo error `th enumber`

Comment (radinamatic) [reply integration_testing/features/admin/admin-find-user.feature:19]: Not important, will correct in future updates!

**Review by mrpau-richard — APPROVED** [review]

**Pull request merged**